pkg/apis/autoscaling/types.go

```diff
@@ -96,20 +96,31 @@ type HorizontalPodAutoscalerSpec struct {
 	// +optional
 	Metrics []MetricSpec
 
-	// Behavior contains settings to specify the scaling behavior both while scaling up
-	// and scaling down. It also is used to customize other aspects of scaling like the
-	// scale down stabilization window
-	// +option
+	// Behaviour contains the scaling behavior for the HPA
+	// If not set, the default behavior is used:
+	//  - for scale up:
+	//    {selectPolicy: "Max", stabilizationWindowSeconds=0, policies: [{"Pods", 4, 60sec}, {"Percent", 100, 60sec}]
+	//	  i.e. either to double the number of pods or to add 4 pods every minute (the largest value is chosen), no stabilization is used
+	//  - for scale down:
+	//    {selectPolicy: "Max", stabilizationWindowSeconds=300, policies: [{"Percent", 100, 60sec}]}
+	//	  i.e. pick the largest recommendation for the last 300 seconds, allow to scale down to 0 pods
+	// +optional
 	Behavior *HorizontalPodAutoscalerBehavior
 }
 
 // HorizontalPodAutoscalerBehavior configures a scaling behavior for Up and Down direction
-// (scaleUp and scaleDown fields respectively)
+// (scaleUp and scaleDown fields respectively). Check HPAScalingRules comment for more info.
```

Comment:
Generally comments should not refer to other comments are should be self-contained. In this case I would said that the comment should say something like _configures the scaling behavior of the target in both up and down directions_.

Reply:
I've changed that already.